src/modules/xiv/rotationBuilder/rotationBuilder.html

```diff
@@ -1,9 +1,23 @@
 <template>
     <main>
+        <script>/* 
         <img src="/assets/recipes-logo.png" alt="logo" />
         <lightning-icon icon-name="action:approval" title="Approved"></lightning-icon>
         <lightning-card icon-name="utility:connected_apps" title="Hello World!"></lightning-card>
         <lightning-button label="I'm a button"></lightning-button>
         <xiv-icon-list action-list={mockActionList} job={job}></xiv-icon-list>
+        */</script>
+        <body>
+            <button id="addGCDButton">Add GCD</button>
+            <button id="addWeaveButton">Add Weave</button>
+            <table>
+              <div class="sortable-list">
```

Comment:
you can inline event binding with a method call like so:
```suggestion
              <div class="sortable-list" ondragstart={dragStart}>
```
Then move the method into the class (the `(e) => ...` portion)

Reply:
I don't completely understand how the drag events work, should I be adding the inline event binding then adding the functions to the DragEvent class or to the sortable-list class?